modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-synchronization_{context}"]
+= Security context constraint synchronization with pod security standards
+
+In addition to the global pod security admission control configuration, {product-title} translates pod security profiles that use the `warn` and `audit` labels for every namespace.
```

Comment:
I don't understand this sentence. What does it translate into what?

The profiles are not using the labels. The labels mark which profiles should be considered during admission.

Reply:
So:

```suggestion
{product-title} applies labels to namespaces to identify which pod security profiles to consider during admission. 
```

?

Reply:
`In addition to the global pod security admission control configuration, {product-title} applies Pod Security Admission Control warn and audit labels to namespaces according to the SCC permissions of the Service Accounts in the given namespace.`

Reply:
```suggestion
In addition to the global pod security admission control configuration, {product-title} applies pod security admission control `warn` and `audit` labels to namespaces according to the SCC permissions of the service accounts that are in a given namespace.
```

Reply:
:+1: